phoenix5-spark/README.md

```diff
@@ -249,14 +312,14 @@ public class PhoenixSparkWriteFromRDDWithSchema {
         }
   
         // Create a DataFrame from the rows and the specified schema
-        df = spark.createDataFrame(rows, schema);
+        Dataset<Row> df = spark.createDataFrame(rows, schema);
         df.write()
             .format("phoenix")
             .mode(SaveMode.Overwrite)
             .option("table", "OUTPUT_TABLE")
             .save();
-  
-        jsc.stop();
+        
```

Comment:
Nit:
I can't see in this editor. Is this extra whitespace ? If yes, then please delete

Reply:
Fixed